reference/constraints/Choice.rst

```diff
@@ -58,6 +58,27 @@ If your valid choice list is simple, you can pass them in directly via the
             protected $genre;
         }
 
+    .. code-block:: php-attributes
+
+        // src/Entity/Author.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Author
+        {
+            const GENRES = ['fiction', 'non-fiction'];
+
+            #[Assert\Choice(["New York", "Berlin", "Tokyo"])]
+            protected $city;
+
+            /**
+             * You can also directly provide an array constant to the "choices" option in the annotation
+             */
+            #[Assert\Choice(choices: Conference::GENRES, message: "Choose a valid genre.")]
```

Comment:
```suggestion
            // You can also directly provide an array constant to the "choices" option in the annotation.
            #[Assert\Choice(choices: Author::GENRES, message: 'Choose a valid genre.')]
```

The code comment explains the following attribute block. I find it confusing to use a phpdoc block for that.

Reply:
Just copy, paste from annotation example. Fixed